src/Services/Azure/Monitor/LogsQueryService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Azure;
+using Azure.Core;
+using Azure.Monitor.Query;
+using Azure.Monitor.Query.Models;
+using AzureMcp.Models.Monitor;
+using AzureMcp.Options;
+using AzureMcp.Services.Interfaces;
+
+namespace AzureMcp.Services.Azure.Monitor
+{
+    internal class LogsQueryService : BaseAzureService, ILogsQueryService
```

Comment:
We have the logs query APIs in `IMonitorService`. Can we use those APIs instead?